docs/en/advanced_guides/datasets.md

```diff
@@ -185,55 +59,27 @@ Then in the config, to use `Filelist` you can modify the config as the following
 ```python
 train = dict(
     type='Filelist',
-    ann_file='image_list.txt',
-    pipeline=train_pipeline
+    ann_file = 'image_list.txt',
+    pipeline=transfrom_list
 )
 ```
```

Comment:
Please use the standard data loader configurations.

````markdown
Then, assume you want to use `Flielist` as the training dataset, you can modify the config file as the following:

```python
train_dataloader = dict(
    ...
    dataset=dict(
        type='Filelist',
        ann_file='image_list.txt',
        pipeline=train_pipeline,
    )
)
```
````